og_migrate/tests/src/Kernel/Migrate/OgUserMembershipD7MigrateTest.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+namespace Drupal\Tests\og_migrate\Kernel\Migrate;
+
+use Drupal\og\Entity\OgMembership;
+use Drupal\og\OgMembershipInterface;
+use Drupal\Tests\migrate_drupal\Kernel\d7\MigrateDrupal7TestBase;
+
+/**
+ * Tests organic group membership migration for users.
+ *
+ * @group og_migrate
+ *
+ * @internal
+ */
+class OgUserMembershipD7MigrateTest extends MigrateDrupal7TestBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public static $modules = [
+    'text',
+    'entity_reference',
+    'telephone',
+    'datetime',
+    'image',
+    'link',
+    'filter',
+    'menu_ui',
+    'comment',
+    'node',
+    'forum',
+    'system',
+    'taxonomy',
+    'user',
+    'og',
+    'og_ui',
+    'og_migrate',
+  ];
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function setUp(): void {
+    parent::setUp();
+
+    $this->loadFixture(__DIR__ . '/../../../fixtures/drupal7.php');
+    $this->installEntitySchema('og_membership_type');
+    $this->installEntitySchema('og_membership');
+    $this->installEntitySchema('og_role');
+    $this->installEntitySchema('user');
+    $this->installEntitySchema('node');
+    $this->installEntitySchema('comment');
+    $this->installEntitySchema('comment_type');
+    $this->installEntitySchema('taxonomy_term');
+    $this->installEntitySchema('taxonomy_vocabulary');
+    $this->installConfig(['comment', 'user', 'node', 'og']);
+    $this->installSchema('comment', ['comment_entity_statistics']);
+    $this->installSchema('forum', ['forum_index', 'forum']);
+    $this->installSchema('node', ['node_access']);
+    $this->executeMigrations([
```

Comment:
We should replace this with helper methods from `Drupal\Tests\migrate_drupal\Kernel\d7\MigrateDrupal7TestBase`

```
    $this->migrateUsers();
    $this->migrateFields();
    $this->migrateTaxonomyTerms();
    ...

```

Reply:
I tried to do this today, but unfortunately the helper methods duplicate migrations and their out of order.